### PR TITLE
Undid edits

### PR DESCRIPTION
**Pull request by nullified33**
## A quick checklist
 - If there's a existing issue, please link to it. If not, provide fill out the same information you would in a normal issue - reproduction steps for bugs, rationale for use-case.
 - If you're working on CraftOS, try to write a few test cases so we can ensure everything continues to work in the future. Tests live in `src/test/resources/test-rom/spec` and can be run with `./gradlew check`.

### CODE REVIEW

File: src/main/resources/assets/computercraft/lua/rom/motd.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-View the source code at https://github.com/mystiacraft/cc-tweaked-fabric
+
+View the source code at https://github.com/3prm3/cc-retweaked
```
Comment (Ueaj-Kerman) [inline src/main/resources/assets/computercraft/lua/rom/motd.txt]: forgot to update that

File: src/main/resources/assets/computercraft/lua/rom/motd.txt [hunk 2]
```diff
@@ -11,3 +12,5 @@ Use "pastebin get" to download a program from pastebin.
 Use "pastebin run" to run a program from pastebin without saving it.
 Use the "edit" program to create and edit your programs.
 You can copy files with "copy" or "cp".
+Run "adventure" to play a text adventure version of Minecraft & run "help" in "adventure" to get more info.
+Please report issues to https://github.com/3prm3/cc-retweaked/issues
```
Comment (Ueaj-Kerman) [inline src/main/resources/assets/computercraft/lua/rom/motd.txt]: and here

File: src/main/resources/data/computercraft/lua/rom/motd.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
-Please report bugs at https://github.com/Zundrel/cc-tweaked-fabric. Thanks!
-View the documentation at https://wiki.computercraft.cc
+Please report bugs at https://github.com/3prm3/cc-retweaked. Thanks!
+View the documentation at https://tweaked.cc
```
Comment (Ueaj-Kerman) [inline src/main/resources/data/computercraft/lua/rom/motd.txt:2]: n here

File: README.md [hunk 4]
```diff
@@ -1,5 +1,47 @@
-# CC:Restitched Patchwork
-# This is a Work In Progress Port
-*it runs and works-ish*
+# ![CC: ReTweaked](logo.png)
```
Comment (Ueaj-Kerman) [inline README.md:1]: should be CC:Restitched

File: README.md [hunk 5]
```diff
@@ -1,5 +1,47 @@
...
+-**Fork the repository:** Make a github account press the fork button.
+
+-**Edit the code** Edit the code and make your changes.
+
+-**Turn on github actions** Press the 'Enable Workflows' button or set up a new gradle workflow and copy-paste [this](https://github.com/3prm3/cc-retweaked/blob/v1.94.2/.github/workflows/gradle.yml) into your worklow's yml, dowload the artifacts and put in your mods folder and check for bugs.
+
+-**Make a pull request** press the 'New Pull Request' button and set the repository on the left to 3prm3/cc-retweaked and set the repository on the right to your repository.
+
+## Bleeding Edge Builds
+Bleeding edge builds can be found [here](https://github.com/3prm3/cc-retweaked/actions) at github actions to simplify things
```
Comment (Ueaj-Kerman) [inline README.md]: change link

**Review by Ueaj-Kerman — CHANGES_REQUESTED** [review]

File: src/main/resources/data/computercraft/lua/rom/motd.txt (hunk 3 above)
Comment (nullified33) [reply src/main/resources/data/computercraft/lua/rom/motd.txt:2]: wiki.computercraft.cc has been phased out

File: README.md (hunk 4 above)
Comment (nullified33) [reply README.md:1]: this is cc-tweaked

**Comment by toad-dev** [comment]
What is the point of this PR? Outside of the texture changes and some string edits to adventure.lua (are those from upstream of just arbitrary changes?), this PR only contains other PRs that have already been merged (from my repo). 

There are a few new commits from Merith-TK's patchwork project that you pulled, but those could be pulled into here more directly when Merith-TK is ready.

**Comment by nullified33** [comment]
> What is the point of this PR? Outside of the texture changes and some string edits to adventure.lua (are those from upstream of just arbitrary changes?), this PR only contains other PRs that have already been merged (from my repo).
> 
> There are a few new commits from Merith-TK's patchwork project that you pulled, but those could be pulled into here more directly when Merith-TK is ready.

Tweaked some stuff in adventure

**Comment by nullified33** [comment]
i probably will edit more stuff later on

**Comment by toad-dev** [comment]
Is it from upstream (the forge CC: Tweaked project)?

**Comment by nullified33** [comment]
no

**Comment by nullified33** [comment]
Upstream from Merith-TK

**Comment by nullified33** [comment]
Also weirldy enough, my readme has no conflicts,
but Merith-TK's does

**Comment by Technici4n** [comment]
Are you applying the updates by hand instead of properly merging the relevant commits?

**Comment by toad-dev** [comment]
Yeah these appear to @Merith-TK commits that are copy pasted and recommitted.

**Comment by nullified33** [comment]
> Yeah these appear to @Merith-TK commits that are copy pasted and recommitted.

recently i undid a lot of them

**Comment by Merith-TK** [comment]
I would request you to stop what you are doing, if you wanted my work added to this repo, you should've requested me to make a PR.

Also, these forks are feature Parity forks. Meaning that they intend to have the features and functions as the mainstream CC:Tweaked.

Any and all changes that are not directly a quilt related bug fix. Are to go to CC:T, not here.

**Comment by Merith-TK** [comment]
Also. I would appreciate it if you would not steal code I have written for CC:Restiched, and simply rebrand it as your own fork. Also I'm pretty sure that violates liscencing, I may be wrong,

It is still morally unethical to take someone else's work and brand it as your own.

**Comment by nullified33** [comment]
> Also. I would appreciate it if you would not steal code I have written for CC:Restiched, and simply rebrand it as your own fork. Also I'm pretty sure that violates liscencing, I may be wrong,
> 
> It is still morally unethical to take someone else's work and brand it as your own.

i undid that code and sorry

**Comment by 3prm3** [comment]
> > Also. I would appreciate it if you would not steal code I have written for CC:Restiched, and simply rebrand it as your own fork. Also I'm pretty sure that violates liscencing, I may be wrong,
> > It is still morally unethical to take someone else's work and brand it as your own.
> 
> i undid that code and sorry for this

**Comment by nullified33** [comment]
https://github.com/3prm3/cc-retweaked/commit/d890bfef0c11cc3581d4bea351f4f95a811956d6

**Comment by nullified33** [comment]
https://github.com/3prm3/cc-retweaked/commit/d5228bde2ff9dbbd9f0e4b8fb2fa72be7070a5ab

**Comment by nullified33** [comment]
ill delete the branch

**Comment by Merith-TK** [comment]
I would also reccomend learning to use git a bit more before attempting a PR like this.

As with git, if your fork is out of date and you made edits. Odds are, you can merge the latest version with your current version, assuming there are no conflicts in the edits.

**Comment by nullified33** [comment]
> I would also reccomend learning to use git a bit more before attempting a PR like this.
> 
> As with git, if your fork is out of date and you made edits. Odds are, you can merge the latest version with your current version, assuming there are no conflicts in the edits.

thanks

**Comment by nullified33** [comment]
> I would also reccomend learning to use git a bit more before attempting a PR like this.
> 
> As with git, if your fork is out of date and you made edits. Odds are, you can merge the latest version with your current version, assuming there are no conflicts in the edits.

good luck with your fork

**Comment by nullified33** [comment]
> I would also reccomend learning to use git a bit more before attempting a PR like this.
> 
> As with git, if your fork is out of date and you made edits. Odds are, you can merge the latest version with your current version, assuming there are no conflicts in the edits.

nah i use desktop,  git i have but is probably outdated or something like that

**Comment by Merith-TK** [comment]
You need to learn how to use git before contributing to a larger project like this with multiple devs.

Otherwise your PR's won't be accepted. Looking at your commits here, this appears to be a clusterfuck of commits.

**Comment by Merith-TK** [comment]
Try using Git Kraken. It works well and is what I use for my git code maintenance

**Comment by nullified33** [comment]
> You need to learn how to use git before contributing to a larger project like this with multiple devs.
> 
> Otherwise your PR's won't be accepted. Looking at your commits here, this appears to be a clusterfuck of commits.

ok

**Comment by nullified33** [comment]
> Try using Git Kraken. It works well and is what I use for my git code maintenance

also ill try it

**Comment by nullified33** [comment]
is it this https://www.gitkraken.com/

**Comment by nullified33** [comment]
> I would request you to stop what you are doing, if you wanted my work added to this repo, you should've requested me to make a PR.
> 
> Also, these forks are feature Parity forks. Meaning that they intend to have the features and functions as the mainstream CC:Tweaked.
> 
> Any and all changes that are not directly a quilt related bug fix. Are to go to CC:T, not here.

pr failed cause of some readme thing, that is why i copied and pasted

**Comment by nullified33** [comment]
also github gave me a "this branch already exsists" error

**Comment by shedaniel** [comment]
please move elsewhere for the discussion, ill be locking this

**Pull request closed without merging**